core/domain/exp_services.py

```diff
@@ -1039,6 +1039,27 @@ def delete_exploration(committer_id, exploration_id, force_deletion=False):
     delete_state_id_mapping_model_for_exploration(
         exploration_id, exploration_version)
 
+    # Remove from subscribers.
+    taskqueue_services.defer(
+        delete_exploration_from_subscribed_users,
+        taskqueue_services.QUEUE_NAME_EVENTS,
```

Comment:
Probably use ONE_OFF_JOBS queue for this -- the events queue is typically for stats events only.